readme.md

```diff
@@ -313,6 +313,14 @@ With the `--postgrest` flag, `gtfs-via-postgres` will augment the schema with a
 
 [read more](docs/postgrest.md)
 
+### Folder mode
+
+With the `--folder-mode` flag, a folder name can be used instead of a file wildcard. All files in the given directory will be read and used in the script. Unsupported files can be ignored using the `-u` flag.
```

Comment:
The `-u` flag does not exist.

Did you LLM-generate this PR? If you did: Please stop doing it, it's really disrespectful!